src/test/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticatorTest.java

```diff
@@ -171,7 +171,7 @@ public void basicTest() throws Exception {
             .put(IDP_METADATA_URL, mockSamlIdpServer.getMetadataUri())
             .put("kibana_url", "http://wherever")
             .put("idp.entity_id", mockSamlIdpServer.getIdpEntityId())
-            .put("exchange_key", "abc")
+            .put("exchange_key", "6aff3042-1327-4f3d-82f0-40a157ac4464")
```

Comment:
why we need this btw?

Reply:
The tests didn't catch the error because they use `abc` as exchange_key. I've updated it to reflect the exchange key used in front-end tests: `6aff3042-1327-4f3d-82f0-40a157ac4464`